packages/fxa-settings/src/lib/sensitive-data-client.ts

```diff
@@ -6,6 +6,20 @@
 
 export const AUTH_DATA_KEY = 'auth';
```

Comment:
fxa-settings/src/pages/Signin/index.tsx has a hard code reference to 'auth'. Maybe update that too?